gradle/publish.gradle

```diff
@@ -36,7 +36,7 @@ def isAvailableForPublication(publication) {
     if (name in ['maven']) return true
 
     def result = false
-    result = result | (name in ['jvm', 'js', 'metadata', 'kotlinMultiplatform'])
+    result = result | (name in ['jvm', 'androidRelease', 'androidDebug', 'js', 'metadata', 'kotlinMultiplatform'])
```

Comment:
We need to add these publications here otherwise they will be filtered out. From the client perspective, the library should be added the following way:

https://github.com/ktorio/ktor-samples/blob/d72b021e2b74fa4ae88c89270bae65b98106a399/app/android-app/build.gradle#L70-L71